packages/cli/internal/pkg/cli/configure_format.go

```diff
@@ -0,0 +1,75 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"github.com/aws/amazon-genomics-cli/internal/pkg/cli/clierror"
+	"github.com/aws/amazon-genomics-cli/internal/pkg/cli/config"
+	"github.com/aws/amazon-genomics-cli/internal/pkg/storage"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+const configureFormatCommand = "configure format"
+
+type formatContextVars struct {
+	format string
+}
+type formatContextOpts struct {
+	formatContextVars
+	configClient storage.ConfigClient
+}
+
+func newFormatContextOpts(vars formatContextVars) (*formatContextOpts, error) {
+	return &formatContextOpts{
+		formatContextVars: vars,
+	}, nil
+}
+
+func (o *formatContextOpts) Validate() error {
+	return nil
+}
+
+// Execute returns a context definition for the specified name.
+func (o *formatContextOpts) Execute() error {
+	err := o.configClient.SetFormat(o.formatContextVars.format)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+func BuildConfigureFormatCommand() *cobra.Command {
+	vars := formatContextVars{}
+	cmd := &cobra.Command{
+		Use:   "format output_format",
+		Short: "Sets default format for output display",
+		Args:  cobra.ExactArgs(1),
+		RunE: runCmdE(func(cmd *cobra.Command, args []string) error {
+			if len(args) == 1 {
```

Comment:
The Args comment on line 48 is forcing this behavior although it also provides a bad error message. To stay consistent with the current package, can you please change Args to "cobra.ArbitraryArgs" and then validating the string passed into the user is both the current number and a correct value. You will need to shuffle around some of the logic below though

Reply:
I noticed with some commands like 'configure describe' and 'configure email', we have cobra.ExactArgs() being used. From what I noticed,  cobra.ExactArgs provides an error message notifying the user. What is the benefit in using cobra.ArbitraryArgs in comparison with the cobra.ExactArgs in this case?

Reply:
The error messages from Cobra are not customer friendly which is why we have started to move to custom messages.

I.e. The following doesn't tell the user what arg it accepts so they need to run `agc configure --help`
```
$ agc configure email
Error: accepts 1 arg(s), received 0
```
vs custom which tells them what is required
```
$ agc context deploy
Error: either an 'all' flag or a list of contexts must be provided, but not both
```